gulpfile.js

```diff
@@ -41,40 +44,34 @@ var configLocal = require('./gulp-config.json'),
 // Installation of components/dependencies
 //
 
-// Bower
-gulp.task('bower', function() {
-  return bower.commands.install()
-    .pipe(gulp.dest(config.bowerPath));
-});
-
 // Web font processing
-gulp.task('fonts', ['bower'], function() {
+gulp.task('move-components-fonts', function() {
   // TODO add custom fonts from components?
-  // gulp.src(config.bowerPath + '/path/to/component/fonts/*/*')
+  // CREATE SEPARATE TASK THAT RETURNS A STREAM:
+  // return gulp.src(config.packagesPath + '/path/to/component/fonts/*/*')
   //   .pipe(gulp.dest(config.dist.fontPath));
 
   // TODO add custom fonts from src directory?
-  // gulp.src(config.src.fontPath + '/path/to/fonts/*')
+  // CREATE SEPARATE TASK THAT RETURNS A STREAM:
+  // return gulp.src(config.src.fontPath + '/path/to/fonts/*')
   //   .pipe(gulp.dest(config.dist.fontPath + '/font-name'));
   return;
 });
 
-gulp.task('bootstrap', ['bower'], function() {
-  gulp.src(config.bootstrap.scss + '/**/*', {base: config.bootstrap.scss})
+// Copy Bootstrap scss files
```

Comment:
It would be nice to be able to reduce some of the file copying that we're doing to accommodate Bower, but I can re-add the bower.json file for now.